src/invoices/InvoicesList/InvoiceLine/InvoiceLineDetailsLayer.js

```diff
@@ -49,21 +61,43 @@ class InvoiceLineDetailsLayer extends Component {
       });
   }
 
+  areTagsEnabled = () => {
```

Comment:
this can be moved to the component with icon

Reply:
Moved